src/utils.js

```diff
@@ -459,10 +460,9 @@ function jsonSize(value) {
 }
 
 function serializeValue(value) {
-  var maxLength = 40;
-
   if (typeof value === 'string') {
-    return value.length <= maxLength ? value : value.substr(0, maxLength - 1) + '\u2026';
+    var maxLength = 40;
```

Comment:
was this change just tangential, or is it part of the fix?

Reply:
(thinking it was just a tangential fix)

Reply:
It was :)